torch/_higher_order_ops/aoti_call_delegate.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# pyre-strict
+
+from __future__ import annotations
+
+import torch
+import torch.utils._pytree as pytree
+from torch._ops import HigherOrderOperator
+from torch._subclasses.fake_tensor import FakeTensor, FakeTensorMode
+from typing import List
+
+
+AOTI_LOWERED_MODULE = "AOTInductorEPModule"
+
+
+class AOTICallDelegate(HigherOrderOperator):
```

Comment:
is it valuable to make this op more general? Like couldn't it also work for MTIA too?